### ISSUE TITLE
Typo in build.gradle

### ISSUE THREAD

**Issue by senox13** [gigaherz/SewingKit#1]
In the manifest section of this project's `build.gradle`, your name is misspelled twice.

```
manifest {
        attributes([
            "Specification-Title": "sewingkit",
            "Specification-Vendor": "gigahrez", <---
            "Specification-Version": "1", // We are version 1 of ourselves
            "Implementation-Title": project.name,
            "Implementation-Version": "${version}",
            "Implementation-Vendor" :"gigahrez", <---
            "Implementation-Timestamp": new Date().format("yyyy-MM-dd'T'HH:mm:ssZ")
        ])
    }
```

**Comment by gigaherz** [comment]
Hah, good catch! Stupid fingers running over each other.

**Comment by gigaherz** [comment]
Fixed in v1.3.0 ;P
